### PR TITLE
Add Stripe testimonial, logos, and url

### PR DESCRIPTION
**Pull request by rosewms**

### CODE REVIEW

File: _data/users.yml [hunk 1]
```diff
@@ -228,6 +228,16 @@
...
+- name: Stripe
+  anchor: stripe
+  logo: /assets/images/logos/stripe-blue.png
+  logosmall: /assets/images/logos/stripe-small.png
+  testimonial: |
+    As of September 3rd 2020, Sigma supports queries using Prestosql v334 SQL
+    syntax. As a part of this change, object (such as map, array, etc) values
+    are now encoded in JSON format.
+  urltext:
+  url: https://stripe.com/
```
Comment (rosewms) [inline _data/users.yml:240]: Should I link to the Stripe support site where they have a continued explanation: https://support.stripe.com/questions/why-has-the-format-of-object-value-in-my-sigma-query-result-changed

**Review by mosabua — CHANGES_REQUESTED** [review]

**Review by mosabua — APPROVED** [review]
Good to go @martint

**Comment by martint** [comment]
Please squash the commits and add a link to https://support.stripe.com/questions/why-has-the-format-of-object-value-in-my-sigma-query-result-changed in a comment for reference, like we do for other entries.

**Comment by rosewms** [comment]
@martint fixed, thanks for your help.

**Pull request merged**